### PR TITLE
ci: make labeler action condition to be more specific on test files

### PR DESCRIPTION
**Pull request by wsuwt**
## Description

Making action labeler to be more specific on test files.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ v7:
 
 documentation:
   - head-branch: ['^doc']
-  - changed-files: '**/*.md'
+  - changed-files: ['**/*.md']
```
Comment (Nantawat-Poothong) [inline .github/labeler.yml:6]: Is it can ignore test files like v6?

Comment (wsuwt) [reply .github/labeler.yml:6]: snapshots in `v7` is under `__test__` and not a markdown files. So I think this condition should be enough.

Comment (Nantawat-Poothong) [reply .github/labeler.yml:6]: I see

**Review by Nantawat-Poothong — APPROVED** [review]
LGTM

File: .github/labeler.yml (hunk 1 above)
Comment (Nantawat-Poothong) [reply .github/labeler.yml:6]: Snapshots on v7 are using `.snap.js` as well, it won't confusing for sure.

**Pull request merged**